data/xml/2020.cl.xml

```diff
@@ -0,0 +1,171 @@
+      <booktitle>Computational Linguistics, Volume 46, Issue 1 - March 2020</booktitle>
+      <month>March</month>
+      <year>2020</year>
+    </meta>
+
+    <paper id="1">
+      <title>On the Linguistic Representational Power of Neural Machine Translation Models</title>
+      <author><first>Yonatan</first><last>Belinkov</last></author>
+      <author><first>Nadir</first><last>Durrani</last></author>
+      <author><first>Fahim</first><last>Dalvi</last></author>
+      <author><first>Hassan</first><last>Sajjad</last></author>
+      <author><first>James</first><last>Glass</last></author>
+      <doi>10.1162/coli_a_00367</doi>
+      <abstract>Despite the recent success of deep neural networks in natural language processing and other spheres of artificial intelligence, their interpretability remains a challenge. We analyze the representations learned by neural machine translation (NMT) models at various levels of granularity and evaluate their quality through relevant extrinsic properties. In particular, we seek answers to the following questions: (i) How accurately is word structure captured within the learned representations, which is an important aspect in translating morphologically rich languages? (ii) Do the representations capture long-range dependencies, and effectively handle syntactically divergent languages? (iii) Do the representations capture lexical semantics? We conduct a thorough investigation along several parameters: (i) Which layers in the architecture capture each of these linguistic phenomena; (ii) How does the choice of translation unit (word, character, or subword unit) impact the linguistic properties captured by the underlying representations? (iii) Do the encoder and decoder learn differently and independently? (iv) Do the representations learned by multilingual NMT models capture the same amount of linguistic information as their bilingual counterparts? Our data-driven, quantitative evaluation illuminates important aspects in NMT models and their ability to capture various linguistic phenomena. We show that deep NMT models trained in an end-to-end fashion, without being provided any direct supervision during the training process, learn a non-trivial amount of linguistic information. Notable findings include the following observations: (i) Word morphology and part-of-speech information are captured at the lower layers of the model; (ii) In contrast, lexical semantics or non-local syntactic and semantic dependencies are better represented at the higher layers of the model; (iii) Representations learned using characters are more informed about word-morphology compared to those learned using subword units; and (iv) Representations learned by multilingual models are richer compared to bilingual models.</abstract>
+      <pages>1–52</pages>
+      <url hash="72e17aed">2020.cl-1.1</url>
+    </paper>
+
+    <paper id="2">
+      <title>The Design and Implementation of XiaoIce, an Empathetic Social Chatbot</title>
```

Comment:
The truecaser hasn't been run?

Reply:
For reference see the call to `normalize` in `ingest.py` (after `from normalize_anth import normalize`).